requirements.txt

```diff
@@ -74,7 +74,8 @@ tensorflow==1.12.0
 tensorflow-estimator==1.10.12
 tensorflow-hub==0.3.0
 termcolor==1.1.0
+tf-sentencepiece==0.1.82
 tqdm==4.31.1
 typing==3.6.6
-urllib3==1.24.2
+urllib3==1.24.1
```

Comment:
why do we downgrade this?

Reply:
aha that was a mismatch with my machine, reverted